static/src/javascripts/projects/common/modules/commercial/create-ad-slot.js

```diff
@@ -102,49 +99,70 @@ define([
         }
     };
 
+    function createAdSlotElement(name, attrs, classes) {
+        return $.create('<div></div>')
+            .attr({
+                'id': 'dfp-ad--' + name,
+                'data-link-name': 'ad slot ' + name,
+                'data-test-id': 'ad-slot-' + name,
+                'data-name': name
+            })
+            .attr(attrs)
+            .addClass('js-ad-slot ad-slot ad-slot--dfp ' + classes.join(' '));
+    }
+
     return function (name, types, series, keywords, slotTarget) {
-        var attrName,
-            slotName = slotTarget ? slotTarget : name,
+        var slotName = slotTarget ? slotTarget : name,
+            attributes = {},
             definition,
-            dataAttrs,
+            classes,
             $adSlot;
 
-        definition = (slotName.match(/^inline/) && slotName !== 'inline1') ? adSlotDefinitions.inline : adSlotDefinitions[slotName];
+        definition = adSlotDefinitions[slotName] || adSlotDefinitions.inline;
```

Comment:
I don't really mind, it just makes spotting issues a little easier if we throw errors rather than fall through to undefined behaviour. But I guess the latter could be nicer on the user.